src/v2/guide/forms.md

```diff
@@ -10,6 +10,8 @@ You can use the `v-model` directive to create two-way data bindings on form inpu
 
 <p class="tip">`v-model` will ignore the initial `value`, `checked` or `selected` attributes found on any form elements. It will always treat the Vue instance data as the source of truth. You should declare the initial value on the JavaScript side, inside the `data` option of your component.</p>
 
+<p class="tip">`v-model` internally uses different properties and emits different events for different input elements. For text and textarea elements it's `value` property and `input` event; for checkboxes and radiobuttons it's `checked` property and `change` event; and for select fields  `v-model` uses `value` as a prop and `change` as an event.</p>
```

Comment:
Two things here:
1) I'm not sure we want to do two tips in a row- we've talked about how we should either join them and use bullets in this case or take one out of tips. In this case, this might be beyond a tip and could be refactored into text with bullet points for each case
2) `'it's` is wrong in this case, we would just use `its` :)

Reply:
@sdras thank you so much for the review! I added a commit to fix mentioned issues